modules/AOW_Conditions/AOW_Condition.php

```diff
@@ -141,4 +141,16 @@ public function save_lines($post_data, $parent, $key = '')
             }
         }
     }
+
+    /**
+     * @return bool
```

Comment:
This function returns `false` as failure because `$this->value_type !== 'Date'` OR `strtolower($value[0]) !== 'anniversary'`? How do you let to know with caller what happend exactly?

Reply:
It's not as failure, it just means it's not a "Date" value type. It could be "Value" or some other valid data. But unless it is "Date" this condition cannot be anniversary type.

Reply:
Ok, but the question is still there if someone does the following:
```
$result = $cond->isDateAnniversaryCondition();
if ($result) {
  echo 'Cool, it is a date anniversary condition :)';
  $this->doSomethingWithIt();
} else {
  echo 'Hi, It is not a date anniversary condition but I have no idea why, maybe it's not date maybe it something else, maybe it's a valid data maybe it's invalid... :(';
  $this->IDontKnowWhatShouldIDo();
}
```